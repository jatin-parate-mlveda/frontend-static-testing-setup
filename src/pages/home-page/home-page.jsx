import { AddBlog, BlogList } from 'features/blog';
import classes from './home-page.module.scss';

export const HomePage = () => (
  <div className={classes.container}>
    <AddBlog />
    <div>
      <BlogList />
    </div>
  </div>
);
