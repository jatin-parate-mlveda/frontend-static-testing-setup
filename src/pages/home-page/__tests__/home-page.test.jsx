import { render } from '@testing-library/react';
import { HomePage } from '../home-page';

test('matches snapshot', () => {
  const { container } = render(<HomePage />);

  expect(container.outerHTML).toMatchSnapshot();
});
