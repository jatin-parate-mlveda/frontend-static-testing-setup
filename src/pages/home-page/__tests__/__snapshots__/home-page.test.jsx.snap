// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `"<div><div class=\\"container\\"><form><div><label for=\\"title\\">Title<input name=\\"title\\" id=\\"title\\" value=\\"\\"></label></div><div><label for=\\"desc\\">Description<input name=\\"desc\\" id=\\"desc\\" value=\\"\\"></label></div><div><button type=\\"submit\\" class=\\" button\\">Submit</button></div></form><div><ul><li>Blog 1</li><li>Blog 2</li><li>Blog 3</li><li>Blog 4</li></ul></div></div></div>"`;
